drivers/SmartThings/zigbee-motion-sensor/src/aqara/init.lua

```diff
@@ -0,0 +1,158 @@
+local capabilities = require "st.capabilities"
+local device_management = require "st.zigbee.device_management"
+local clusters = require "st.zigbee.zcl.clusters"
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+local cluster_base = require "st.zigbee.cluster_base"
+local data_types = require "st.zigbee.data_types"
+
+local sensitivityAdjustment = capabilities["stse.sensitivityAdjustment"]
+local detectionFrequency = capabilities["stse.detectionFrequency"]
+local sensitivityAdjustmentId = "stse.sensitivityAdjustment"
+local detectionFrequencyId = "stse.detectionFrequency"
+local sensitivityAdjustmentCommand = "setSensitivityAdjustment"
+local detectionFrequencyCommand = "setDetectionFrequency"
+
+local PowerConfiguration = clusters.PowerConfiguration
+local OccupancySensing = clusters.OccupancySensing
+
+local PRIVATE_CLUSTER_ID = 0xFCC0
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+local MFG_CODE = 0x115F
+local MOTION_TEMP_ATTRIBUTE_ID = 0x0112
+local FREQUENCY_ATTRIBUTE_ID = 0x0102
+local SENSITIVITY_ATTRIBUTE_ID = 0x010C
+local FREQUENCY_DEFAULT_VALUE = 120
+local UNOCCUPIED_TIMER = "unoccupiedTimer"
+local FREQUENCY_PREF = "frequencyPref"
+local SENSITIVITY_HIGH = 3
+local SENSITIVITY_MEDIUM = 2
+local SENSITIVITY_LOW = 1
+local OCCUPANCY_OCCUPIED = 1
+
+local FINGERPRINTS = {
+  { mfr = "LUMI", model = "lumi.motion.agl02" },
+  { mfr = "LUMI", model = "lumi.motion.agl04" }
+}
+
+local is_aqara_products = function(opts, driver, device)
+  for _, fingerprint in ipairs(FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+      return true
+    end
+  end
+  return false
+end
+
+local write_motion_pref_attribute = function(device, cluster, attr, value)
+  device:send(cluster_base.write_manufacturer_specific_attribute(device, cluster, attr, MFG_CODE,
+    data_types.Uint8, value))
+end
+
+local function added_handler(self, device)
+  device:emit_event(capabilities.motionSensor.motion.inactive())
+  device:emit_event(sensitivityAdjustment.sensitivityAdjustment.Medium())
+  device:emit_event(detectionFrequency.detectionFrequency(FREQUENCY_DEFAULT_VALUE))
+  device:emit_event(capabilities.battery.battery(100))
+end
+
+local do_configure = function(self, device)
+  device:configure()
```

Comment:
It isn't maybe necessary. Please test after remove this with real device. (also need to change the TC)

Reply:
Thanks for the suggestion.